framework/build.gradle

```
@@ -94,72 +87,58 @@ android {
     }
 }
 
-install {
-    repositories.mavenInstaller {
-        pom {
-            project {
-                packaging 'aar'
-                name 'Cordova'
-                url 'https://cordova.apache.org'
+task sourcesJar(type: Jar) {
+    from android.sourceSets.main.java.srcDirs
+    classifier = 'sources'
+}
+
+publishing {
+    publications {
+        Cordova(MavenPublication) {
+            groupId = 'org.apache.cordova'
+            artifactId = 'framework'
+            version = '10.0.0-dev'
+
+            artifact(sourcesJar)
+
+            pom {
+                name = 'Cordova'
+                description = 'A library to build Cordova-based projects for the Android platform.'
+                url = 'https://cordova.apache.org'
+
                 licenses {
                     license {
-                        name 'The Apache Software License, Version 2.0'
-                        url 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+                        name = 'Apache License, Version 2.0'
+                        url = 'https://www.apache.org/licenses/LICENSE-2.0.txts'
```

```suggestion
                        url = 'https://www.apache.org/licenses/LICENSE-2.0.txt'
```
invalid url, has an extra s

---

Thanks for noticing this. I have commited your suggestion.